### PR TITLE
Tweak SSS transmission

### PR DESCRIPTION
**Pull request by EvgeniiG**

### CODE REVIEW

File: Assets/ScriptableRenderPipeline/HDRenderPipeline/Material/Lit/Lit.hlsl [hunk 1]
```diff
@@ -753,10 +756,9 @@ void EvaluateBSDF_Directional(  LightLoopContext lightLoopContext,
 
     [branch] if (bsdfData.enableTransmission)
     {
-        float LdotV = dot(L, V); // Also computed in BSDF()
-        // Compute the normal at the back of the object as R = reflect(N, -V)
-        // float RdotL = NdotL - 2 * preLightData.NdotV * LdotV;
-        float illuminance = saturate(-LdotV);
+        // Reverse the normal + do some wrap lighting to have a nicer transition between regular lighting and transmittance
+        // Ref: Steve McAuley - Energy-Conserving Wrapped Diffuse
+        illuminance = saturate((-NdotL + SSS_WRAP_LIGHT) / ((1 + SSS_WRAP_LIGHT) * (1 + SSS_WRAP_LIGHT)));
```
Comment (JulienIgnace-Unity) [inline Assets/ScriptableRenderPipeline/HDRenderPipeline/Material/Lit/Lit.hlsl]: Maybe factorize this code? It's duplicated in each EvaluateBSDF_xxx

File: Assets/ScriptableRenderPipeline/HDRenderPipeline/Material/Lit/SubsurfaceScatteringProfile.cs [hunk 2]
```diff
@@ -44,6 +46,7 @@ public enum TransmissionMode : uint { None = SssConstants.SSS_TRSM_MODE_NONE, Th
         public SubsurfaceScatteringProfile()
         {
             surfaceAlbedo      = Color.white;
```
Comment (JulienIgnace-Unity) [inline Assets/ScriptableRenderPipeline/HDRenderPipeline/Material/Lit/SubsurfaceScatteringProfile.cs]: I guess this one could be removed?

Comment (JulienIgnace-Unity) [inline Assets/ScriptableRenderPipeline/HDRenderPipeline/Material/Lit/SubsurfaceScatteringProfile.cs]: I guess this one is useless now?

Comment (EvgeniiG) [inline Assets/ScriptableRenderPipeline/HDRenderPipeline/Material/Lit/SubsurfaceScatteringProfile.cs]: It's still in use. :-)

File: Assets/ScriptableRenderPipeline/HDRenderPipeline/Material/Lit/Lit.hlsl (hunk 1 above)
Comment (EvgeniiG) [inline Assets/ScriptableRenderPipeline/HDRenderPipeline/Material/Lit/Lit.hlsl]: Done.

**Pull request merged**